LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-requestSubmit-expected.txt

```diff
@@ -1,3 +1,6 @@
-FAIL: Timed out waiting for notifyDone to be called
 
-{"error": {"code": 404, "message": ""}}
+
+Harness Error (TIMEOUT), message = null
+
+TIMEOUT <form> requestSubmit() sets sourceElement Test timed out
```

Comment:
While it is good to generate the new result, we may as well skip this test in TestExpectations. When I did my PR to more carefully check whether sourceElement is supported in the first place, I probably omitted this one. Looking now, I see no point in the test if the whole focus of it is sourceElement (which we do not support atm).